src/components/DashboardHome/DashContent.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Tabs } from 'antd';
+import './Dashboard.css';
+import Pane1 from './Pane1';
+import './Dashboard.css'
```

Comment:
Do we need the second import of Dashboard.css?

Reply:
Sorry - my mistake. Resolved.